src/platform/packages/shared/kbn-visualization-ui-components/package.json

```diff
@@ -2,8 +2,5 @@
   "name": "@kbn/visualization-ui-components",
   "private": true,
   "version": "1.0.0",
-  "license": "Elastic License 2.0 OR AGPL-3.0-only OR SSPL-1.0",
-  "sideEffects": [
-    "*.scss"
-  ]
```

Comment:
You should change this to `sideEffects: false` to avoid a different treeshake in the bundle.
sideEffects is by default true